iOS14UIKitTour/iOS14UIKitTour/Models/Repository.swift

```diff
@@ -1,6 +1,14 @@
 
-struct Repository {
+struct Repository: Codable {
     let name: String
-    let id: String
-    let contributors: [Contributor]
+    let user: Contributor
```

Comment:
A repository only has a single a contributor? How does that work?

Reply:
I think I see how the API works. A repo has an owner, then you can get the list of contributors to a repo with the repo name and the owner name.
https://docs.github.com/en/rest/reference/repos#list-repository-collaborators
`/repos/{owner}/{repo}/collaborators`

Reply:
yep!